spring-data-couchbase/src/main/java/org/springframework/data/couchbase/core/convert/DateConverters.java

```diff
@@ -204,7 +205,7 @@ public enum NumberToDateTimeConverter implements Converter<Number, DateTime> {
 
 		@Override
 		public DateTime convert(Number source) {
-			return source == null ? null : new DateTime(source.longValue());
+			return source == null ? null : new DateTime(source.longValue(), DateTimeZone.UTC);
```

Comment:
This change is to make the Couchbase deserialization compatible with the SDK (via jackson-jodatime) which is in UTC (vs. local time zone).